.eslintrc.yml

```diff
@@ -0,0 +1,32 @@
+env:
+  es6: true
+  mocha: true
+  node: true
+
+extends:
+  - eslint:recommended
+  - plugin:node/recommended
+
+plugins:
+  - node
+
+root: true
+
+rules:
+  node/exports-style: [error, module.exports]
+  node/no-unpublished-require: [error, {allowModules: [chai, jsdom]}]
```

Comment:
I'm hitting the no-unpublished-require rule with a new package now (`diff`). Do you understand what makes that happen? I don't see anything from https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/no-unpublished-require.md.